frontend/src/scenes/session-recordings/playlist/sessionRecordingsListLogic.ts

```diff
@@ -323,25 +332,20 @@ export const sessionRecordingsListLogic = kea<sessionRecordingsListLogicType>([
 
     urlToAction(({ actions, values, props }) => {
         const urlToAction = (_: any, params: Params, hashParams: HashParams): void => {
-            if (!props.updateSearchParams) {
-                return
-            }
             const nulledSessionRecordingId = hashParams.sessionRecordingId ?? null
             if (nulledSessionRecordingId !== values.selectedRecordingId) {
                 actions.setSelectedRecordingId(nulledSessionRecordingId)
             }
 
             const filters = params.filters
-            if (filters) {
+            if (filters && props.updateSearchParams) {
```

Comment:
Noticed that we weren't persisting the hash param which we always want to for every playlist